bazel/repository_locations.bzl

```diff
@@ -119,8 +119,8 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Aspect Bazel helpers",
         project_desc = "Base Starlark libraries and basic Bazel rules which are useful for constructing rulesets and BUILD files",
         project_url = "https://github.com/aspect-build/bazel-lib",
-        version = "1.3.1",
-        sha256 = "4ef2f746bae7bd7f1ec39dc9b53a9d7e8002f18233ea2c2ee4702bbb5283c7ca",
+        version = "1.7.0",
+        sha256 = "58fab45b4419a5eaa340489d567a013e80334c8e4bd998288149cd7e39437586",
```

Comment:
you need to adjust the date but otherwise lgtm

Reply:
ah crap always gets me

Reply:
thanks done

Reply:
for ref this should work

```console

$ export GITHUB_TOKEN=$(my.secrets)
$ bazel run //tools/dependency:check -- -c release_dates

```

Reply:
the secret doesnt need any power - just gh access